perf-tool/src/monitor.cpp

```diff
@@ -215,3 +215,302 @@ JNIEXPORT void JNICALL MonitorContendedEntered(jvmtiEnv *jvmtiEnv, JNIEnv *env,
         }
     }
 }
+JNIEXPORT void JNICALL MonitorContendedEnter(jvmtiEnv *jvmtiEnv, JNIEnv *env, jthread thread, jobject object){
+    json j;
+    jvmtiError error;
+    
+    /* Get number of events and increment */
+    int numSamples = atomic_fetch_add(&monitorSampleCount, 1);
```

Comment:
This can stay like this for now, but in the future, if we want to track the time the monitor is held, we need to change it.
The reason is that two events, "monitor_enter" and "monitor_entered" are using the same `monitorSampleCount` and if I get a sample for "monitor_enter" I may not get a sample for the corresponding "monitor_entered"